boards/launchxl/src/ccfg.rs

```diff
@@ -10,11 +10,11 @@
 #[used]
 #[link_section = ".init"]
 pub static CCFG_CONF: [u32; 22] = [
-    0x01800000, 0xFF820010, 0x0058FFFD, 0xF3FFFF3A, 0xFFFFFFFF, 0xFFFFFFFF, 0xFFFFFFFF, 0xFFFFFFFF,
+    0x01800000, 0xFF820010, 0x0058FFFC, 0xF3FBFF3A, 0xFFFFFFFF, 0xFFFFFFFF, 0xFFFFFFFF, 0xFFFFFFFF,
```

Comment:
what's this do?
might be worth abstracting CCFG a little bit so it's not these raw bytes values, but not within scope of this PR